local_ads/local_ads_tests/statistics_tests.cpp

```diff
@@ -0,0 +1,162 @@
+#include "local_ads/statistics.hpp"
+#include "testing/testing.hpp"
```

Comment:
Usually, in tests, testing.hpp should be included first. So, from my point of view, the order should be:
```c++
#include "testing/testing.hpp"

#include "local_ads/statistics.hpp"

#include "coding/file_name_utils.hpp"
```